RELEASE-NOTES.md

```diff
@@ -14,6 +14,7 @@
 * Empty strings are now detected as invalid language codes in the term classes
 * Made the `Item` constructor parameters optional
 * Made the `Fingerprint` constructor parameters optional
+* Made the `Property` constructor parameter for fingerprint nullable
```

Comment:
@JeroenDeDauw: So instead of answering my question you did this https://github.com/wmde/WikibaseDataModel/commit/1a9577ff102137de52033c36efa423ccf079c35b, which is incorrect from my perspective, since the parameter is not "optional". You can not skip the parameter and  call `new Property( new PropertyId( 'P1' ), 'string' )`.

What now? I don't want to revert your change without an agreement.